translate/translate.py

```diff
@@ -4,39 +4,89 @@
 
 LANG = {'ko': 'en', 'en': 'ko'}
 
+try:
+    pyperclip.paste()
+    CLIPBOARD_MESSAGE = 'Using clipboard data.'
+except pyperclip.PyperclipException:
+    CLIPBOARD_MESSAGE = """
+    This error should only appear on Linux (not Windows or Mac). 
+    You can fix this by installing one of the copy/paste mechanisms:
+    
+    sudo apt-get install xsel to install the xsel utility.
```

Comment:
It seems to depend on a specific platform? 
If everything is not going to work out, would not it be better to encourage users to install it?